packages/editor/src/components/post-schedule/index.js

```diff
@@ -45,19 +45,17 @@ export default function PostSchedule( { onClose } ) {
 				after: getDayOfTheMonth( previewedMonth ),
 				before: getDayOfTheMonth( previewedMonth, false ),
 				exclude: [ select( editorStore ).getCurrentPostId() ],
+				per_page: 100,
+				_fields: 'id,date',
```

Comment:
@GeoJunkie I know we've been discussing the performance of this component, and I thought I should confirm that restricting the fields we retrieve like this solves the problems we've been seeing, as it stops the values for the other fields being calculated.

Reply:
I just tested the latest update on a test site and the performance is much better now. Thanks @pablinos !